src/main/java/com/example/shopproject/entity/Basket.java

```diff
@@ -0,0 +1,37 @@
+package com.example.shopproject.entity;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "t_basket")
+public class Basket {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private Long userID;
+    private Long product_id;
+
+    public Long getId() {
```

Comment:
if you have a lombok, then use it)

Reply:
u dont need getter setter constructors equals hashcode toString written on you own, read about lombok